client/src/containers/ProductDetailContainer.tsx

```diff
@@ -47,6 +49,7 @@ const ProductDetailContainer = (props: IProductDetailContainer): JSX.Element =>
       onBuyBtnClick={onBuyBtnClick}
       onCartBtnClick={onCartBtnClick}
       onWishBtnClick={onWishBtnClick}
+      isWish={wishStatus.data?.isWished ? true : false}
```

Comment:
```suggestion
      isWish={wishStatus.data?.isWished ?? false}
```
이런 식으로 표현도 가능할 것 같아요!
data가 없을 경우 false로 넣어주고, 있으면 isWished의 값을 씁니다!
17 라인에서 `{ data, refetch }` 로 구조 분해해서 받아도 좋을 것 같기도 하네요 ☺️

Reply:
오 좋습니다
17번 라인에 대해서 현재 사용하고있는 useQuery가 두개이기 때문에 둘 다구조분해를 하긴 힘들 것 같아서 서로 일관성있게 구조분해를 하지않고 받는걸로 하였습니다 :)

Reply:
앗 하나가 더 있었군요. 고걸 못봤네요 ㅎ헣